pucauto.py

```
@@ -249,10 +258,13 @@ def build_trades_dict(soup):
     trades = {}
 
     for row in soup.find_all("tr", id=lambda x: x and x.startswith("uc_")):
+        member_points = int(row.find("td", class_="points").text)
+        if member_points < CONFIG["min_value"]:
```

Fail fast if this member doesn't have enough points.